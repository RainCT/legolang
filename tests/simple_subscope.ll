if true: print "hi"

for i in 1 .. 3: print i

string[] s
s[] = "all"
s[] = "ok"
foreach str in s: print str
