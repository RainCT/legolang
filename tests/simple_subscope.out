hi
1
2
3
all
ok
